docs/conceptual/iommu.rst

```diff
@@ -0,0 +1,98 @@
+.. meta::
+   :description: Input-Output Memory Management Unit (IOMMU)
+   :keywords: IOMMU, DMA, PCIe, XGMI, AMD, ROCm
+
+****************************************************************
+Input-Output Memory Management Unit (IOMMU)
+****************************************************************
+
+Overview
+================================================================
+
+The IOMMU (IO Memory Management Unit) is a device which provides memory remapping services to 
+IO devices on a platform. Similar to the MMU on a CPU, the IOMMU is able to provide a private 
+virtual address space for each device or group of devices on a platform. While the IOMMU is 
+most commonly used for virtualization provide a private IO space for devices used in a guest 
+virtual machine, it can also be used to isolate devices on bare metal to prevent accidental
+or nefarious DMA accesses to things that they should should not have access to. The IOMMU provides
```

Comment:
What things exactly?

Reply:
I believe things refer to the protected memory space

Reply:
replace "things" with "system addresses (memory or MMOIO space allocated to other devices or users"

Reply:
s/MMOIO/MMIO/